contracts/contracts/trees/AccQueue.sol

```diff
@@ -269,22 +266,20 @@ abstract contract AccQueue is Ownable, Hasher {
     return depth;
   }
 
-  /**
-   * Merge all subtrees to form the shortest possible tree.
-   * This function can be called either once to merge all subtrees in a
-   * single transaction, or multiple times to do the same in multiple
-   * transactions. If _numSrQueueOps is set to 0, this function will attempt
-   * to merge all subtrees in one go. If it is set to a number greater than
-   * 0, it will perform up to that number of queueSubRoot() operations.
-   * @param _numSrQueueOps The number of times this function will call
-   *                       queueSubRoot(), up to the maximum number of times
-   *                       is necessary. If it is set to 0, it will call
-   *                       queueSubRoot() as many times as is necessary. Set
-   *                       this to a low number and call this function
-   *                       multiple times if there are many subroots to
-   *                       merge, or a single transaction would run out of
-   *                       gas.
-   */
+  /// @notice Merge all subtrees to form the shortest possible tree.
+  /// This function can be called either once to merge all subtrees in a
+  /// single transaction, or multiple times to do the same in multiple
+  /// transactions. If _numSrQueueOps is set to 0, this function will attempt
+  /// to merge all subtrees in one go. If it is set to a number greater than
```

Comment:
I think "this function will attempt to merge all subtrees in one **operation**" would be clearer for me.
== update ==
after reading the param's description, I think if the _numSrQueueOps is set to 0, it will call queueSubRoot() as many times as is necessary? I think the descriptions are conflicted.

Reply:
yea these descriptions are quite old and admit I didn't really try to change it but agree they should be refreshed! I'll do that now :)

Reply:
so I think it actually makes sense, what is meant by one go is one transaction, not one `queueSubRoot`.
On the other hand if you specify a number != 0 it will then only perform n `queueSubRoot` and not necessarily complete the merging. hope this is a bit clearer now

Reply:
Regardless, I removed the description you flagged and keeping the one on the param to avoid confusion